templates/common/infra/terraform/applicationinsights.tf

```diff
@@ -0,0 +1,1241 @@
+terraform {
+  required_providers {
+    azurerm = {
+      version = "~>3.18.0"
+      source  = "hashicorp/azurerm"
+    }
+    azurecaf = {
+      source  = "aztfmod/azurecaf"
+      version = "1.2.15"

```

Comment:
Just wondering - do we want to have some sort of range for this constraint or is fixing to a single version (unlike the azurerm provider) what we want?